server/routing/game.go

```diff
@@ -13,27 +18,54 @@ const (
 
 // GameRoutes contains information about routes specific to game interaction
 type GameRoutes struct {
-	Service services.GameService
+	Services *services.MasterList
 }
 
 // AddRoutes will add all game routes to the given router
 func (gr *GameRoutes) AddRoutes(r *mux.Router) {
-	r.HandleFunc(gameRoute+"/{gameName}/tick/{tick}/player/{playerName}/cards", gr.submitCardsHandler).Methods("PUT")
-	r.HandleFunc(gameRoute+"/{gameName}/tick", gr.getCurrentTickHandler).Methods("GET")
-	r.HandleFunc(gameRoute+"/{gameName}/tick/{tick}/player/{playerName}", gr.getGameStateHandler).Methods("GET")
+	r.HandleFunc(gameRoute+"/{gameID}/turn/{turn}/player/{playerName}", gr.submitCardsHandler).Methods("PUT")
+	r.HandleFunc(gameRoute+"/{gameID}/turn", gr.getCurrentTurnHandler).Methods("GET")
+	r.HandleFunc(gameRoute+"/{gameID}/turn/{turn}/player/{playerName}", gr.getGameStateHandler).Methods("GET")
 }
 
-// submitCardsHandler handles a player request to submit cards for the given tick
+// submitCardsHandler handles a player request to submit cards for the given turn
 func (gr *GameRoutes) submitCardsHandler(w http.ResponseWriter, r *http.Request) {
-	panic("Not yet implemented")
+	w.Write([]byte("Success, though nothing has been recorded"))
 }
 
-// getCurrentTickHandler returns the current tick for the requested game
-func (gr *GameRoutes) getCurrentTickHandler(w http.ResponseWriter, r *http.Request) {
-	panic("Not yet implemented")
+// getCurrentTurnHandler returns the current turn for the requested game
+func (gr *GameRoutes) getCurrentTurnHandler(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	gameID := vars["gameID"]
+	//submissionTurn := vars["turn"]
+
+	gameUUID, err := uuid.FromString(gameID)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("%v is not a valid game ID", gameID), http.StatusBadRequest)
+		return
+	}
+
+	game, err := gr.Services.Game.GetActiveGame(gameUUID)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("failed to find game with id '%v'", gameID), http.StatusNotFound)
```

Comment:
After thinking about it, we might want to reserve StatusNotFound for when there is no matching route.  I think typical web servers use 404 for a non-matching route and possibly a 400 when it is something like a non-matching id.